### PR TITLE
week 7 grammar

### PR DESCRIPTION
**Pull request by lhmartin**
- Updated the readmes to improve the English grammar 
- Expanded on some areas explanations to be closer to the lectures

### CODE REVIEW

File: docs/en/week07/07-1.md [hunk 1]
```diff
@@ -64,59 +64,61 @@ This allows a machine to produce multiple outputs, not just one.
...
 
-$$P(y \mid x) = \frac{\exp (-\beta F(x,y))}{\int_{y'}\exp(-\beta F(x,y'))}$$ 
+$$P(y \mid x) = \frac{e^{-\beta F(x,y)}}{\int_{y'}e^{-\beta F(x,y')}}$$
+
 
 where $\beta$ is positive constant and needs to be calibrated to fit your model. Larger $\beta$ gives a more fluctuate model while smaller $\beta$ gives a smoother model. (In physics, $\beta$ is inverse temperature: $\beta \rightarrow \infty$ means temperature goes to zero).
 
-$$P(y,z \mid x) = \frac{\exp(-\beta F(x,y,z))}{\int_{y}\int_{z}\exp(-\beta F(x,y,z))}$$
+
+$$P(y,z \mid x) = \frac{e^{-\beta F(x,y,z)}}{\int_{y}\int_{z}e^{-\beta F(x,y,z)}}$$
```
Comment (Atcold) [inline docs/en/week07/07-1.md]: Nope.

File: docs/en/week07/07-1.md [hunk 2]
```diff
@@ -64,59 +64,61 @@ This allows a machine to produce multiple outputs, not just one.
...
-\begin{aligned}
-P(y \mid x) & = \frac{\int_z \exp(-\beta E(x,y,z))}{\int_y\int_z \exp(-\beta E(x,y,z))} \\
-& = \frac{\exp \left [ -\beta \left (-\frac{1}{\beta}\log \int_z \exp(-\beta E(x,y,z))\right ) \right ] }{\int_y \exp\left [ -\beta\left (-\frac{1}{\beta}\log \int_z \exp(-\beta E(x,y,z))\right )\right ]} \\
-& = \frac{\exp (-\beta F_{\beta}(x,y))}{\int_y \exp (-\beta F_{\beta} (x,y))}
-\end{aligned}
+<center>
+<img src="{{site.baseurl}}/images/week07/07-1/fig2.png"/><br>
+<b>Fig. 2</b>: Derivation for marginalization in probabilistic models
+
+</center>
```
Comment (Atcold) [inline docs/en/week07/07-1.md]: NOPE!

File: docs/en/week07/07-1.md [hunk 3]
```diff
@@ -64,59 +64,61 @@ This allows a machine to produce multiple outputs, not just one.
...
-$$
 
 Thus, if we have a latent variable model and want to eliminate the latent variable $z$ in a probabilistically correct way, we just need to redefine the energy function $F_\beta$ (Free Energy)
 
 
 ### Free Energy
 
 $$
-F_{\beta}(x,y) = - \frac{1}{\beta}\log \int_z \exp (-\beta E(x,y,z))
+F_{\beta}(x,y) = - \frac{1}{\beta}\log \int_z e^{-\beta E(x,y,z)}
```
Comment (Atcold) [inline docs/en/week07/07-1.md]: Nope!

File: docs/en/week07/07-1.md [hunk 4]
```diff
@@ -64,59 +64,61 @@ This allows a machine to produce multiple outputs, not just one.
...
 
 <center>
-<img src="{{site.baseurl}}/images/week07/07-1/fig3.png" height="75%" width="75%"/><br>
-<b>Fig. 2</b>: Visualization of the energy function that captures dependency between x and y
+<img src="{{site.baseurl}}/images/week07/07-1/fig3.png"/><br>
+<b>Fig. 3</b>: Visualization of the energy function that captures dependency between x and y
```
Comment (Atcold) [inline docs/en/week07/07-1.md]: Why?

File: docs/en/week07/07-1.md [hunk 5]
```diff
@@ -64,59 +64,61 @@ This allows a machine to produce multiple outputs, not just one.
...
 
 <center>
-<img src="{{site.baseurl}}/images/week07/07-1/fig4.png" height="75%" width="75%"/><br>
-<b>Fig. 3</b>: Example for multiple prediction of EBM as an implicit function
+<img src="{{site.baseurl}}/images/week07/07-1/fig4.png"/><br>
+<b>Fig. 4</b>: Example for multiple prediction of EBM as an implicit function
```
Comment (Atcold) [inline docs/en/week07/07-1.md]: Again?

**Comment by lhmartin** [comment]
I’m sorry Alf, I thought I had updated to the latest version, but what probably happened is I wasn’t at the right version so my version was making changes I didn’t actually intend. These changes were by no means intentional. I’ll be more careful next time.

**Comment by Atcold** [comment]
Can you please rebase on top of `master`?
Here in the PR, you can switch forth and back the source and the destination.
Let me know if you need assistance.

File: docs/en/week07/07-2.md [hunk 6]
```diff
@@ -179,20 +173,27 @@ If the DAE is properly trained, the energy grows quadratically as we move away f
...
 <b>Fig. 10</b>: Contrastive Divergence
 </center>
+
+
+1. What are some advantages of latent variable EBMs?
+   - The main advantage of Latent variable models is that they allow multiple predictions through the latent variable.
+2. Is K-means an architectural or a contrastive method?
+   -  Architectural as all we do is push the energy down in certain regions
+3. Give one example of where you would use a Denoising AE?
+   - BERT
```
Comment (Atcold) [inline docs/en/week07/07-2.md]:
```suggestion
```

**Comment by Atcold** [comment]
I've converted a file from `dos` to `unix`.
That's why it looks like it's a completely new file.
I think I've managed to preserve all your edits, while moving the file to the most recent version of the repo.

**Pull request merged**